src/client/java/io/wispforest/limelight/impl/ui/ResultsContainerComponent.java

```diff
@@ -58,11 +63,17 @@ private void rebuildContents() {
             clearChildren();
 
             for (var entry : results) {
-                var result = new ResultEntryComponent(screen, entry);
+                boolean isExpanded = entry instanceof ExpandableResultEntry && expanded.getOrDefault(entry.entryId(), false);
```

Comment:
i kind of don't like relying on `entryId` here...
this could work, but imo it should also be documented in the `#entryId` method that it affects this too